google/resource_compute_network.go

```diff
@@ -143,6 +190,55 @@ func resourceComputeNetworkRead(d *schema.ResourceData, meta interface{}) error
 	return nil
 }
 
+func resourceComputeNetworkUpdate(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+
+	project, err := getProject(d, config)
+	if err != nil {
+		return err
+	}
+
+	network := d.Get("name").(string)
+
+	d.Partial(true)
+
+	if d.HasChange("peering") {
```

Comment:
We have a lot of code dedicated to working with changing a set of peerings, but our tests only have 1 or 0 on each network. Let's test that code by adding a test that has several peerings, and updates keeping some but removing/adding others.